lib/tenstorrent/jtag_bootrom/jtag_bootrom.c

```diff
@@ -4,13 +4,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include "blackhole_offsets.h"
-
 #include <stdint.h>
 
 #include <tenstorrent/bh_chip.h>
+#include <tenstorrent/control_reg.h>
```

Comment:
Could we maybe just copy the register definitions into the jtag workaround, since it's (hopefully) a 1-off thing? Otherwise, it's a bit unusual to share platform-specific headers between two different FW domains.